features/step_definitions/register_steps.rb

```diff
@@ -0,0 +1,9 @@
+Then("I should be redirected to {string} page") do |page_name|
+  expect(page.current_path).to eq page_path(page_name)
+end
```

Comment:
We have a step in basic_steps that do the same thing. 
```
Then("I should be on the {string} page") do |article_title|
  expect(page.current_path).to eq find_article(article_title)
end
``` 

Please refactor test and remove this